### ISSUE TITLE
Recursively search folders for potential mods

### ISSUE THREAD

**Issue by Alex-Au1** [nhok0169/Anime-Game-Remap#30]
## Story
This feature ( https://github.com/nhok0169/Fix-Raiden-Boss/pull/28 ) proved that there is no difference in fixing a `merged.ini` and a `.ini` file for an individual mod. So the original structure for merged mod (folders of individual mods being the direct children of the folder for the merged mod) does not hold.

## Change
To find which potential mods to fix next, after fixing `foobar.ini` file in folder `foo`:

- Recursively search all children folders from `foo`
- Recursively search all children folders from each resource in `foobar.ini` that the function `Inifile.fix` references

Since all `.ini` files are now the same, need to handle case of multiple nested commands:

_**eg.**_
```ini
[TextureOverrideRaidenShogunBlend]
run = CommandListRaidenShogunBlend
...

[CommandListRaidenShogunBlend]
if $swapvar == 1
   run = SubCommandBlend1
else
   run = SubCommandBlend2
endif
...

[SubCommandBlend1]
...

[SubCommandBlend2]
...

```

Haven't seen any mods using the above syntax yet, but should still be valid in the modified .ini language GIMI uses.

<br>
(man, I really should make a proper compiler at this point :pensive: ... Good thing this language highly relies on newline characters)

**Comment by nhok0169** [comment]
just do a normal base for the code export from merge script.
if they make a custom raiden merge mod they need do the fix for them mod.
or you can contact @SilentNightSound and make a new merge script support raiden fix automatic without this raiden fix 😉

**Comment by Alex-Au1** [comment]
hehe... i'm just using issues as a backlog for now, so going to work on the change when I' am less busy

**Comment by Alex-Au1** [comment]
So the above is not really an issue ( cause nobody found out about it yet! )

Just a cool feature that I want to do :yum:

**Comment by Alex-Au1** [comment]
In development...

**Comment by Alex-Au1** [comment]
**Completed:** https://github.com/nhok0169/Fix-Raiden-Boss/pull/31
